packages/web/src/pages/sign-up-page/pages/CreateEmailPage.tsx

```diff
@@ -205,11 +206,7 @@ export const CreateEmailPage = () => {
               >
                 {createEmailPageMessages.signUpMetamask}
               </Button>
-              <ConnectedMetaMaskModal
-                open={isMetaMaskModalOpen}
-                onBack={() => setIsMetaMaskModalOpen(false)}
-                onSuccess={() => navigate(SIGN_UP_HANDLE_PAGE)}
-              />
+              <ExternalWalletSignUpModal />
```

Comment:
Annoyingly still needs to be in `CreateEmailPage` despite being portaled - I think the rest of the app is unmounted so it couldn't go in the standard modal provider?